php/src/Google/Protobuf/Internal/MapFieldIter.php

```diff
@@ -117,17 +120,18 @@ public function key()
      *
      * @return void
      */
+    #[\ReturnTypeWillChange]
     public function next()
```

Comment:
```suggestion
    public function next(): void
```
According to doc, any return value will be ignored, so it seems to be safe to use "void" return type hint